nncf/torch/quantization/layers.py

```diff
@@ -186,19 +186,30 @@ def reset_call_counter(self):
     def get_trainable_params(self) -> Dict[str, torch.Tensor]:
         raise NotImplementedError
 
-    def apply_minmax_init(self, min_values, max_values, log_module_name: str = None):
+    def apply_minmax_init(self,
+                          min_values: torch.Tensor,
+                          max_values: torch.Tensor,
+                          log_module_name: str = None):
         """min_values and max_values must have the same shape as specified in self.scale_shape"""
         if self.initialized:
             nncf_logger.debug("Skipped initializing {} - loaded from checkpoint".format(log_module_name))
             return
+
+        if torch.all(torch.isinf(min_values)) or torch.all(torch.isinf(max_values)):
+            raise AttributeError('Statistics are not collected for {}'.format(log_module_name))
```

Comment:
I think it is better to use `ValueError` here and below according to error [description](https://docs.python.org/3/library/exceptions.html#ValueError).

Reply:
Makes sense, done.